src/main/java/org/cloudfoundry/promregator/rewrite/MergableMetricFamilySamples.java

```diff
@@ -63,7 +63,7 @@ public void merge(HashMap<String,MetricFamilySamples> others) {
 		
 		for (Iterator<MetricFamilySamples> iterator = coll.iterator(); iterator.hasNext();) {
 			MetricFamilySamples mfs = iterator.next();
-			if (mfs.type == Type.UNTYPED) {
+			if (mfs.type == Type.UNKNOWN) {
```

Comment:
With the Prometheus SImpleCLient upgrade `UNTYPED` went away so I replaced it with `UNKNOWN`